horace_core/sqw/axes_blocks/@line_axes/private/convert_old_struct_into_nbins_.m

```diff
@@ -27,3 +27,8 @@
     in.label = in.ulabel;
     in = rmfield(in,'ulabel');
 end
+if sum(nbins_all_dims) == 4
```

Comment:
Slightly cryptic way of saying 
```
if all(nbins_all_dims == 1)
```

Reply:
well, probably its alternative way of doing this.